backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -42,17 +44,25 @@
   @Override
   public IGitRebaseParameters getParametersForRebaseOntoParent(BaseGitMacheteNonRootBranch branch)
       throws GitMacheteMissingForkPointException {
+    log.debug("Enter getParametersForRebaseOntoParent for ${branch.getName()}");
     var forkPoint = branch.getForkPoint();
     if (forkPoint.isEmpty()) {
       throw new GitMacheteMissingForkPointException("Cannot get fork point for branch '${branch.getName()}'");
     }
 
     var newBaseBranch = branch.getUpstreamBranch();
+
+    log.debug(
+        "Inferred rebase parameters: currentBranch = ${branch.getName()}, newBaseCommit = ${newBaseBranch.getPointedCommit().getHash()}, forkPointCommit = ${forkPoint.get().getHash()}");
```

Comment:
Maybe instead break the log string somewhere in the middle with `..." + "...` - here and in other similar places